sdk/helper/automatedrotationutil/fields.go

```diff
@@ -104,3 +112,76 @@ func AddAutomatedRotationFields(m map[string]*framework.FieldSchema) {
 		m[name] = schema
 	}
 }
+
+// HandleRotationJobOperation is a helper method to register and deregister rotation jobs
```

Comment:
Can we add something here about how this function (in this version anyway) is a direct synthesis of the other two functions and the rollback func?